mongodb/intro/query-operators/query-collection-4.md

```diff
@@ -5,7 +5,7 @@ levels:
   - beginner
 
 aspects:
-  - introduction
+  - new
```

Comment:
don't think the `new` aspect is a good fit:
> new: Updates to the topic or subtopic, recent features or changes to an API.

https://github.com/enkidevs/curriculum/wiki/Insight-Documentation#aspects

Reply:
Not sure

Reply:
so the `$gt` and all other concpts found in this insight:
- are not an update to the topic (MongoDB) because they were in the languange since the start
- are not a recent feature
- are not a change to the API

We should change the `new` aspect with one that fits better (such as `workout`)